### ISSUE TITLE
[css] disable autocomplete of property value

### ISSUE THREAD

**Issue by svipas** [Microsoft/vscode#68495]
I can't find how to disable CSS property value autocomplete, I'm using these settings:

```json
  "editor.quickSuggestions": false,
  "editor.suggestOnTriggerCharacters": false,
  "editor.tabCompletion": "on",
```

it works perfectly in JavaScript, TypeScript, etc. But in CSS if I press tab I will get full autocomplete of property and then value. How I can disable autocomplete of value and keep only autocomplete of property? Gif below shows the issue.

![css](https://user-images.githubusercontent.com/10441177/52617279-f535c700-2ea3-11e9-9634-a131abdf9789.gif)

As you can see it autocompleted almost perfectly except it automatically opened suggestion for value which I don't need since I want to go through all `w` based suggestions with `tab` and I even added in settings `"editor.quickSuggestions": false` which CSS should respect.

In my opinion, if `quickSuggestions` is `false` it should autocomplete only with `white-space` and then you can go through properties by pressing `tab` and then you add `:` which results to `white-space: ` you can write a single letter and go through suggestions again with `tab`.

So to summarize it should autocomplete with only `white-space` and do not show suggestion list of values for that prop.

**Comment by octref** [comment]
> since I want to go through all `w` based suggestions with `tab`

How does that work? Can't you use arrow keys to select prev/next item?

**Comment by svipben** [comment]
No, only `tab` or `shift+tab`. I proposed this suggestion https://github.com/Microsoft/vscode/issues/68557, if any member of your team agrees on that proposal, then this can be closed, because I really miss suggestions list, but it's too noisy and distracting. Since there are a really small group of devs who disable suggestions and uses only tab to autocomplete, also in CSS, chances this will be re-created by someone else is very very small.

**Comment by octref** [comment]
@aeschli  The problem is:

- He want to use tab/S-tab to go prev/next
- Some CSS completion items `editor.action.triggerSuggest` and trap the tab/S-tab

Do you think we should add the setting?

**Comment by svipas** [comment]
@aeschli @octref Maybe I can help somehow to make this happen?

**Comment by aeschli** [comment]
Sure, a PR that adds a new setting is welcome.

**Comment by svipas** [comment]
@aeschli So this is related to `editor.action.triggerSuggest`?

**Comment by octref** [comment]
Sure, thanks! Here are some pointers:

- Dev setup is here: https://github.com/Microsoft/vscode/blob/master/extensions/css-language-features/CONTRIBUTING.md
- Add a boolean setting `css.completion.triggerPropertyValueCompletion` to toggle this behavior on/off
- Here's the logic for re-triggering value completion after property completion: https://github.com/Microsoft/vscode-css-languageservice/blob/27c789a065207a89a80541ede1e7561a72c97cd1/src/services/cssCompletion.ts#L186-L190
- Update the test suite

**Comment by svipas** [comment]
@octref Thank you!
